### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by jkmaswell**
## Summary

Adding navbar to project.

## Screenshots

image
image

## Trello Card

https://trello.com/c/Xnk5LyGq/13-navbar

### CODE REVIEW

File: scss/navbar.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  .navbar-logo,
+  .navbar-title {
+    padding: 0 12px;
+  }
+
+  .navbar-logo {
+    max-height: 30px;
+  }
+}
```
Comment (FrankIglesias) [inline scss/navbar.scss]: add tailing line

File: scss/navbar.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 20px -3px $black;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  min-width: 100%;
+  position: fixed;
+  z-index: 2;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: why 2?

File: scss/navbar.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 20px -3px $black;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  min-width: 100%;
+  position: fixed;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: define top and right or left values for this class

File: scss/navbar.scss (hunk 1 above)
Comment (jkmaswell) [reply scss/navbar.scss]: done

File: scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.navbar {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 0 0 20px -3px $black;
+  display: flex;
+  height: 90px;
+  justify-content: space-between;
+  left: 0;
+  min-width: 100%;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: width 100%;

**Pull request merged**